lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -211,6 +215,7 @@ def add_address(form, options)
         billing_address = options[:billing_address] || options[:address]
 
         if billing_address
+          form[:avs_address]    = truncate(billing_address[:address1], 30)
```

Comment:
This was added back in by Adam in a previous commit, so I may just need to re-rebase.

Reply:
Yep, another rebase cleared this up! :D